engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/subscription-access-mode.directive.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @ngdoc directive
+ * @name Bastion.subscriptions:subscriptionAccessMode
+ *
+ * @requires contentAccessMode
+ *
+ * @description
+ *   Component for showing information about content access mode (whether content is
+ *   allowed with or without a subscription)
+ */
+angular.module('Bastion.subscriptions').directive('subscriptionAccessMode',
```

Comment:
Also, can you please write a little test for this.  I know this behavior itself isn't complex but if we ever add any complexity to this the inertia to write tests for it will be more easily overcome.